Firestore/core/src/local/leveldb_mutation_queue.cc

```diff
@@ -126,9 +126,10 @@ BatchId LoadNextBatchIdFromDb(DB* db) {
 
 LevelDbMutationQueue::LevelDbMutationQueue(const User& user,
                                            LevelDbPersistence* db,
+                                           LevelDbIndexManager* index_manager,
```

Comment:
Do we need LevelDbIndexManager here? We can remove the cast if we just pass in IndexManager.

Reply:
Done.